helpers/event_helper.rb

```diff
@@ -0,0 +1,11 @@
+module EventHelper
+  def event_time(evt_start, evt_end = nil, mobile_only: false)
```

Comment:
I'll go with `event_start`. I'd rather have `start` but `end` is a reserved keyword.